apps/version.c

```diff
@@ -42,29 +42,21 @@ const OPTIONS version_options[] = {
     {"b", OPT_B, '-', "Show build date"},
     {"d", OPT_D, '-', "Show configuration directory"},
     {"e", OPT_E, '-', "Show engines directory"},
+    {"m", OPT_M, '-', "Show modules directory"},
```

Comment:
There is no such thing in 1.1.1...

Reply:
right, and it just prints N/A but I see that as a useful feature for scripts,
instead of having to figure out which version it is,